lib/src/deploy.rs

```diff
@@ -747,6 +747,9 @@ pub(crate) async fn rollback(sysroot: &Storage) -> Result<()> {
     } else {
         println!("Next boot: rollback deployment");
     }
+
+    sysroot.update_mtime()?;
```

Comment:
Unrelated to this particular changed line but I think we could test this in one of the integration tests pretty easily.

Reply:
:+1: good thing too because while writing the test I just realized i accidentally swapped `version` and `digest`!

Reply:
Thinking about punting this to a followup.  Unless I'm blind, it doesn't look like in integration we currently test any of upgrade/edit/switch/rollback so that's a whole other thing.

Reply:
Ah yeah I was just looking at the stuff under `tests-integration`